packages/@aws-cdk/aws-ec2-alpha/test/integ.route-v2.ts

```diff
@@ -84,9 +80,13 @@ const eigw = new EgressOnlyInternetGateway(stacks.eigw, 'testEOIGW', {
 });
 new Route(stacks.eigw, 'testEIGWRoute', {
   routeTable: routeTables.eigw,
-  destination: '0.0.0.0/0',
+  destination: '10.0.0.0/24',
   target: { gateway: eigw },
 });
+const eigwAddRoute = new EgressOnlyInternetGateway(stacks.eigw, 'testEOIGWviaAddRoute', {
```

Comment:
i think we don't need to add another eigw, and can just add route with a new destination to earlier eigw. Let me know if this is not correct.

Reply:
Tested it, and it seems like you're right: it doesn't fail if we make a new route with the same EIGW. Pushing new changes now 👍